openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,15 +109,17 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None):
+def get_programs_with_type(types=None, include_hidden=True):
```

Comment:
No, I don't think so. This function is only called from home page and courses page.